pages/common/mosquitto_pub.md

```diff
@@ -0,0 +1,27 @@
+# mosquitto_pub
+
+> A simple MQTT version 3.1.1 client that will publish a single message on a topic and exit.
+
+- Publish temperature value of 32 on `sensors/temperature` topic to localhost with `QoS` 1:
```

Comment:
Perhaps this will read better?

```suggestion
- Publish a temperature value of 32 on the `sensors/temperature` topic to localhost with `QoS` 1:
```

What does `QoS` stand for? Can we expand it at all if it's an abbreviation?